frontend/src/layouts/ProtectedLayout.vue

```diff
@@ -6,16 +6,30 @@ import SideNav, {
     type ISideNavItem,
 } from '@/components/common/SideNav.vue';
 import sideNavData from '@/static/sideNav.json';
-import { isApplicationSelected } from '@/store/ApplicationState';
+import { isApplicationSelected,selectedApplicationShortDisplayText } from '@/store/ApplicationState';
 
 const navigationData = ref<[ISideNavData]>(sideNavData as any);
 
+// Show and hide the correct sideNav btn based on the application
+const sideNavAddPermissionOption = () => {
+    console.log('test')
+    if(selectedApplicationShortDisplayText.value === 'FAM') {
```

Comment:
let's switch to use the constant variable

Reply:
did it, It is now using the constant